tools/myst/host/exec.c

```diff
@@ -289,6 +303,22 @@ int exec_action(int argc, const char* argv[], const char* envp[])
          */
         cli_getopt(&argc, argv, "--app-config-path", &commandline_config);
 
+        /* Get --symlink options (there may be more than one) */
+        {
+            const char* arg = NULL;
+
+            symlinks[0] = NULL;
+
+            if ((cli_getopt(&argc, argv, "--symlink", &arg) == 0))
```

Comment:
Do we still need the ``--symlink`` option?

Reply:
I think we can hide that option since it is only used internally, and --symlink currently doesn't check if LINK is defined in config.json.